src/mlpack/methods/ann/rbm/spike_slab_rbm_impl.hpp

```diff
@@ -116,49 +113,49 @@ RBM<InitializationRuleType, DataType, PolicyType>::Phase(
   }
 
   spikeBiasGrad = spikeMean;
-
-  visiblePenaltyGrad = -0.5 * arma::dot(input, input)
-      / std::pow(input.n_cols, 2);
+  // Setting visiblePenaltyGrad.
+  gradient.row(weightGrad.n_elem + spikeBiasGrad.n_elem) = -0.5 * arma::dot(
+                                    input, input) / std::pow(input.n_cols, 2);
```

Comment:
```suggestion
       input, input) / std::pow(input.n_cols, 2);
```

This should be tabbed twice (4 spaces).